onnxruntime/core/optimizer/conv_add_act_fusion.cc

```diff
@@ -287,12 +281,16 @@ class FuseConvAddActivationAction : public ReplaceWithNew {
 void RegisterConvAddActivationFusionRules(SelectorActionRegistry& registry) {
   auto action = std::make_unique<actions::FuseConvAddActivationAction>();
   auto selector = std::make_unique<selectors::ConvAddActivationSelector>();
-  registry.RegisterSelectorAndAction("ConvAddAct", {{"Conv", {1, 11}}},
+  SelectorActionRegistry::OpVersionsMap op_versions_map;
+  const std::string kMSDomainFusedConv = std::string(kMSDomain) + ":FusedConv";
+  const std::string kMSInternalNHWCDomainNhwcFusedConv = std::string(kMSInternalNHWCDomain) + ":NhwcFusedConv";
```

Comment:
If this is not being used for the JS EP yet and there are no unit tests we should exclude the changes from this PR. 

e.g. I believe NhwcFusedConv is in the contrib ops kMSDomain so using kMSInternalNHWCDomain may not match anything and you won't get the expected optimization. we should have unit tests to cover this to validate it works correctly.

Reply:
Reverted changes to conv_add_act_fusion.cc